Gemfile

```diff
@@ -2,11 +2,19 @@ source 'https://rubygems.org'
 
 ruby '2.5.0'
 
-gem 'rake'
-gem 'rubocop', '0.56.0'
+gem 'sinatra'
+gem 'sinatra-flash'
+gem 'bcrypt'
+gem 'pg'
+gem 'simplecov'
+gem 'simplecov-console'
 
```

Comment:
In order to timestamp your posts you can use `gem timecop`. [timecop link](https://github.com/travisjeffery/timecop) for a bit of info